src/transformers/utils/hub.py

```diff
@@ -657,13 +649,12 @@ def _create_repo(
                     repo_id = repo_id.split("/")[-1]
                 repo_id = f"{organization}/{repo_id}"
 
-        token = HfFolder.get_token() if use_auth_token is True else use_auth_token
         url = create_repo(repo_id=repo_id, token=token, private=private, exist_ok=True)
 
         # If the namespace is not there, add it or `upload_file` will complain
         if "/" not in repo_id and url != f"{HUGGINGFACE_CO_RESOLVE_ENDPOINT}/{repo_id}":
             repo_id = get_full_repo_name(repo_id, token=token)
-        return repo_id, token
+        return repo_id
```

Comment:
This is used in all `save_pretrained` methods, so you will need to update all of those if you change the signature (search for `_create_repo` in the repo, all the hits are for this method).

Reply:
Ok thanks for the feedback. I made the update in https://github.com/huggingface/transformers/pull/21333/commits/086aaaee8c67d6f4e12583890eccd9ad4e7e502b to adapt all subclasses of `PushToHubMixin`.

I also reverted the argument name change (back to `use_auth_token`) as I saw it was called in a lot of cases using `**kwargs` so it would mean changing user behavior.